.travis.yml

```diff
@@ -19,7 +21,11 @@ install:
   - conda create -q -n test-env python=$TRAVIS_PYTHON_VERSION pip coverage nose sphinx numpydoc flake8
   - source activate test-env
   - if [[ $DEP_VERSIONS == "oldest" ]]; then
-      conda install numpy==1.7.0 scipy==0.11.0 sympy==0.7.4.1 cython==0.17 theano==0.6.0;
+      if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then
+        conda install numpy==1.7.0 scipy==0.11.0 sympy==0.7.4.1 cython==0.17 theano==0.6.0;
+      else
+        conda install numpy==1.8.1 scipy==0.13.3 sympy==0.7.5 cython==0.20.1 theano==0.7.0;
+      fi
```

Comment:
Ok, that looks good. I'll open an issue to build some Python 3 packages for the combination used in 2.7 so that we can have a common min dependencies in the future.